app/modules/inventory/lib/parsers/common.coffee

```diff
@@ -7,28 +9,29 @@ module.exports = (data)->
     delete data.isbn
 
   if authors?
-    data.authors = deduplicateAndJoinAuthors authors
+    data.authors = deduplicateAuthors authors
 
   return data
 
-deduplicateAndJoinAuthors = (authors)->
+deduplicateAuthors = (authors)->
   return authors
   # Prevent names containing comma to pass as they will later be interpretted
   # as several names
   .filter containsNoComma
   .filter isFirstOccurence(authors)
-  .join ', '
 
 containsNoComma = (str)-> str? and not /,/.test(str)
 
 isFirstOccurence = (array)-> (str, index)->
   noramlizedStr = normalize str
   for previousStr in array[0...index]
-    if noramlizedStr is normalize(previousStr) then return false
+    if leven(noramlizedStr, normalize(previousStr)) < 3 then return false
```

Comment:
does this number have any reason behind its definition (aka why 3?), maybe would be nice to comment this line with the explanation ?

Reply:
arbitrary number documented in dacabf82